Cargo.toml

```diff
@@ -1,7 +1,12 @@
 [package]
 name = "vss-client"
-version = "0.1.0"
+version = "0.0.1-alpha"
```

Comment:
@tnull Wonder if we should stick to "0.1.0" or "0.1.0-alpha" given experience with `rust-lightning` vs `ldk-node`. Thoughts?